src/plugin.spec.ts

```diff
@@ -1013,4 +1013,100 @@ describe('OIDC plugin (local OIDC provider)', function () {
       validateToken(getJWTContents(result.accessToken));
     });
   });
+
+  describe('with stubbed oidc client', function () {
+    const sandbox = sinon.createSandbox();
+    const plugin = new MongoDBOIDCPluginImpl({});
+    const client = {
+      userinfo: sandbox.stub(),
+      introspect: sandbox.stub(),
+      revoke: sandbox.stub(),
+    };
```

Comment:
It would be really nice to have some form of proper integration test here, I can try to look into that later. Usually if the `OIDCTestProvider` has some specific, surprising behavior, there’s a good reason for that.

Reply:
🤷 this doesn't happen with Okta and nothing in the documentation for this oidc library mentions that token would be non-active by default, feel free to take a look

Reply:
Hm yeah, it looks like the `oidc-provider` package doesn’t do userinfo for JWT access tokens. :confused: 

You could still use this test provider for introspect and revoke, introspection requires an `introspection: { enabled: true },` entry in the `features` list but otherwise it seems to work.

Or, if you like, you could expand `OIDCMockProvider` to support this and use that, like we do in `integration.spec.ts`, if you want an integration test for these methods.

Reply:
Hmmm, I had tests for introspection initially, but this always returns `active: false` for the issued tokens, so adding tests for revoking was not really doing anything that I could meaningfully test, this is when I dropped this idea completely 🤔 Was it working for you?

Reply:
I think that fails similarly because this test provider doesn’t actually store info for JWTs?

Reply:
Yeah, probably! I'm mostly double-checking that you are seeing the same behavior